pkg/asset/manifests/infrastructure.go

```diff
@@ -76,7 +76,6 @@ func (i *Infrastructure) Generate(dependencies asset.Parents) error {
 			InfrastructureName:   clusterID.InfraID,
 			APIServerURL:         getAPIServerURL(installConfig.Config),
 			APIServerInternalURL: getInternalAPIServerURL(installConfig.Config),
-			EtcdDiscoveryDomain:  getEtcdDiscoveryDomain(installConfig.Config),
```

Comment:
And i think we need to also update the API https://github.com/openshift/api/blob/master/config/v1/types_infrastructure.go#L67 to be `//+optional`

Reply:
Created a PR to make EtcdDiscoveryDomain optional:

https://github.com/openshift/api/pull/767